_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -109,11 +93,26 @@ This `iframe` element does not have a `tabindex` [attribute value][] that is a n
 <iframe tabindex="0" srcdoc="<a href='/'>Home</a>"></iframe>
 ```
 
+#### Inapplicable Example 2
+
+This `iframe` element contains no content that is part of [sequential focus navigation][].
```

Comment:
The Applicability does not mention "content that is part of sequential focus navigation".
So this example is currently Applicable as it is an `iframe` with negative tabindex, and it passes the Expectation as there is no focusable content in it.

Reply:
I have switched the expectation and applicability around to help make these valid.